front/src/main/scss/main.scss

```diff
@@ -138,38 +130,36 @@
 }
 
 .select {
-
   margin-top: 1.8rem;
   padding-left: 1rem;
 }
 
 .coll {
-  margin-top:3.5rem;
-  float:left;
-  display:table-cell;
-  vertical-align:middle;
-  width:2rem;
+ margin-top: 5.35rem;
+ float: left;
+ margin-right: 3rem;
+ margin-left: 0.6rem;
 }
+
 .coll button {
-  display:table;
-  text-indent:1em;
-  white-space:nowrap;
-  transform:rotate(90deg) translate( -50%,0);
-  transform-origin:0.6em center;
+  width: 2rem;
+  height: 3rem;
+  padding-bottom: 0.2rem;
+  text-indent: -0.5rem;
 }
-.collapse1 {
-  display:none;
+
+.typeSelect {
+  background-color: transparent;
 }
 
-#sideButton {
-  clear:none;
-  display:block;
+.divselector {
+  display:inline-block;
 }
 
-.typeSelect{
-  background-color: transparent;
-  heigth:16px;
+.glyphicon-option-vertical {
+  padding-bottom:0.5rem;
 }
-.divselector{
-    display:inline-block;
+
+.btnS{
```

Comment:
you could add a more specific class name like `creation-button` or more specific selector rule, with a generic class name, like .btn-success.creation (don't forget to make other rules related to `creation` more specific as well)